### ISSUE TITLE
Compass Plugin not working

### ISSUE THREAD

**Issue by activestylus** [padrino/padrino-framework#449]
I have not had any luck whatsoever using compass with Padrino

```
padrino g project myapp -e haml -c compass
```

The above generates all the files, including lib/compass-plugin.rb. But when I try to run 

```
compass watch
```

_Nothing to compile. If you're trying to start a new project, you have left off the directory argument.  
Run "compass -h" to get help._

So I try to run it explicitly, specifying the stylesheet like so:

```
compass watch app/stylesheets/application.scss
```

_No such file or directory - myapp/src_

I assumed that compass watch command was not properly integrated into padrino and continued on my merry way using plain old sass --watch, until I tried to do something very simple in my stylesheet:

```
@import compass
```

_File to import not found or unreadable: compass_

In short, nothing appears to work as it should. Tried this on several fresh apps using Ruby 1.8.7 and 1.9.2, but no love
